plugins/utils/signatureutils.go

```diff
@@ -92,10 +97,14 @@ func signaturesToCommands(signatures []*components.PluginSignature) []cli.Comman
 			Usage:           sig.Usage,
 			SkipFlagParsing: true,
 			Action: func(c *cli.Context) error {
-				return gofrogcmd.RunCmd(
+				output, err := gofrogcmd.RunCmdOutput(
 					&SignatureCmd{
 						sig.ExecutablePath,
 						c.Args()})
+				if err == nil {
+					log.Output(output)
```

Comment:
We probably don't want this output as debug only.

Reply:
I am not sure I understand what you meant here. The purpose of this log is to set the output of the plugin command, as output of the CLI process, so users will be able to catch it's output.